Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:6
+
+RUN npm install -g \
+      bower \
```

Comment:
Add `npm@5` here to be sure we use npm version 5. Default is 3.x.x AFAIK.

Reply:
[this](https://stackoverflow.com/a/44280995) is fun. As the stackoverflow controbutor puts it: "¯\_(ツ)_/¯".

Basically since npm<5 is broken I'll need to use yarn to update npm@5. Luckily yarn is already in the node docker image.

Reply:
Done in 0dcb81aaa2cccd0d4f9a94fbf4ee7d877d5d2cd4